src/pages/resultsView/ResultsViewPageStoreUtils.ts

```diff
@@ -250,3 +250,10 @@ export async function fetchQueriedStudies(filteredPhysicalStudies:{[id:string]:C
 
     return queriedStudies;
 }
+
+
+export function isExpressionProfile(profileId:string, version:number): boolean {
```

Comment:
I dont think this function is well-named, it doesnt just distinguish an expression profile it distinguishes an rna seq expression profile.. @jjgao confirm?